docs/integrations/optuna.rst

```diff
@@ -3,150 +3,214 @@
 Neptune-Optuna Integration
 ==========================
 
-This integration enables you to monitor |Optuna| hyperparameter optimization in Neptune.
+|Run on Colab|
 
-.. image:: ../_static/images/integrations/optuna_neptuneai.png
-   :target: ../_static/images/integrations/optuna_neptuneai.png
-   :alt: Optuna Neptune integration
+What will you get with this integration?
+----------------------------------------
 
-Requirements
-------------
-To use Neptune + Optuna integration you need to have installed is |neptune-client| and |neptune-contrib|.
+|optuna-tour|
 
-.. code-block:: bash
+|Optuna| is an open source hyperparameter optimization framework to automate hyperparameter search. With Neptune integration, you can:
 
-    pip install neptune-client neptune-contrib['monitoring']
+* see the experiment as it is running,
+* see charts of logged run scores,
+* log the parameters tried at every run,
+* log the best parameters after training,
+* explore interactive optuna visualizations like plot_contour, plot_slice, plot_parallel_coordinate, optimization_history,
```

Comment:
log interactive optuna visualizations like plot_contour, plot_slice, plot_parallel_coordinate, and optimization_history